generator/main.ts

```diff
@@ -104,13 +102,11 @@ async function main([root_dir, out_dir, ...idl_files]) {
   });
 
   var env = new nunjucks.Environment();
-  env.addFilter('camelcase', function(str, count) {
-    return str.replace(/(?:^\w|[A-Z]|\b\w|\s+)/g, function(match, index) {
-      return index == 0 ? match.toUpperCase() : match;
-    });
+  env.addFilter('pascalcase', function(str, count) {
```

Comment:
Is this possible?
`env.addFilter('pascalcase', changeCase.pascalCase);`
(In javascript, I'm sure it works but I'm not sure in TypeScript)

Reply:
it is working well.
so i replaced it.